modules/tinymce/src/core/test/ts/browser/dom/RangePointTest.ts

```diff
@@ -28,7 +28,7 @@ UnitTest.asynctest('browser.tinymce.core.dom.RangePointsTest', (success, failure
         sAssertXYWithinRange(15, 5)
       ])),
       Logger.t('point in table selection', GeneralSteps.sequence([
-        tinyApis.sSetContent('<p><table><tbody><tr><th>Header 1</th><th>Header 2</th></tr><tr><td>Cell 1</td><td>Cell 2</td></tr></tbody></table></p>'),
+        tinyApis.sSetContent('<p><table><tbody><tr><th>Heading 1</th><th>Heading 2</th></tr><tr><td>Cell 1</td><td>Cell 2</td></tr></tbody></table></p>'),
```

Comment:
This probably shouldn't have been changed, as it's referring to a row header instead of a `h1` etc... case.

Reply:
Not a massive issue either way, as it's only a test.

Reply:
```suggestion
        tinyApis.sSetContent('<p><table><tbody><tr><th>Header 1</th><th>Header 2</th></tr><tr><td>Cell 1</td><td>Cell 2</td></tr></tbody></table></p>'),
```